serum.go

```diff
@@ -16,17 +16,30 @@ type Injector struct {
 	envVars        *envparser.EnvVars
 }
 
+// NewInjector creates a new injector using the provided options.
+func NewInjector(options ...Option) (*Injector, error) {
```

Comment:
I thought we agreed to have the `from` as required param `NewInjector(from fInterface, options ...Option)`. otherwise the API is bit confusing and error prone

Reply:
Oh, I must have misunderstood. That works too. I currently have them as Options. I could make a separate type for the source.

Reply:
Updated! The first param now is an interface called a Loader. It's responsible for loading env vars from an arbitrary source.